polyfill/lib/duration.mjs

```diff
@@ -123,9 +123,11 @@ export class Duration {
     if (!dateParts.length && !timeParts.length) return 'PT0S';
     return `P${dateParts.join('')}${timeParts.join('')}`;
   }
-  toLocaleString(...args) {
+  toLocaleString() {
+    // TODO fix this once Intl.DurationFormat has a working polyfill (or a spec).
     if (!ES.IsDuration(this)) throw new TypeError('invalid receiver');
-    return new Intl.DateTimeFormat(...args).format(this);
+    console.error('This function requires Intl.DurationFormat.');
+    return this.toString();
```

Comment:
As Ms2ger suggested on the Absolute.toJSON PR I'd recommend moving the bulk of `toString()` into `ES.DurationToString` and calling it here as well.

Reply:
I did it despite you pointing this out, because I was planning to revisit this function later, but your comment at https://github.com/tc39/proposal-temporal/pull/465#discussion_r396797015 is spot on, so I'll address these two together.